concordium-sdk/src/main/java/com/concordium/sdk/responses/Fraction.java

```diff
@@ -54,4 +57,29 @@ public String toString() {
     public double asDouble() {
         return (double) this.numerator.getValue() / (double) this.denominator.getValue();
     }
+
+    /**
+     * Get the fraction as a {@link BigDecimal} value with the specified amount of precision.
+     * Result is rounded using {@link RoundingMode#HALF_UP}
```

Comment:
It's unclear to me what this "precision" means from this documentation.

An example or two or explanation would help.

Reply:
I added a more descriptive comment.